database/db-demos/src/db/demos/persistence/PlayerRepo.java

```diff
@@ -0,0 +1,16 @@
+package db.demos.persistence;
+
+import java.util.List;
+
+import jakarta.data.repository.By;
+import jakarta.data.repository.CrudRepository;
+import jakarta.data.repository.Find;
+import jakarta.data.repository.Repository;
+
+@Repository(dataStore = "league")
+public interface PlayerRepo extends CrudRepository<Player, Integer> {
+
+  @Find
+  List<Player> findByLastName(@By("lastName") String lastName);
```

Comment:
The first example should be as simple as possible. Then introduce more annotations in the following methods. According to the specification @Find and @By should not be necessary here.

Reply:
@By is optional, but without the @Find the repository codegen is silently skipped.
Passing `-verbose` as compiler flag exposes the issue:
```
[INFO]  + [10 units compiled]
[INFO]  + [compiled 784 lines in 706 ms: 1110.4 lines/s]
[INFO]  + [10 .class files generated]
[WARNING] 1. INFO: Hibernate compile-time tooling 7.0.3.Final
2. WARNING in /mnt/data/dev/repos/demo-projects/database/db-demos/src/db/demos/persistence/PlayerRepo.java (at line 11)
	List<Player> findByLastName(String lastName);
	             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
repository method cannot be implemented (skipping whole repository)
----------
2 problems (0 errors, 1 warning, 1 info)

[INFO]  + [validated processes in 253 ms ]
```

Reply:
ahh good. Then we can be sure that this is not hibernate-processor specific. Thanks for double-checking :+1: